src/content/roadmap/statelessness/index.md

```diff
@@ -0,0 +1,100 @@
+- compatible with future upgrades to Ethereum's cryptography
+
+Read more on [why it's so important to go stateless](https://dankradfeist.de/ethereum/2021/02/14/why-stateless.html)
+Read the original [stateless client concept notes](https://ethresear.ch/t/the-stateless-client-concept/172)
+
+### Weak Statelessness {#weak-statelessness}
+
+Weak statelessness does involve changes to the way Ethereum nodes verify state changes, but it does not completely eliminate the need for state storage in all nodes on the network. Instead, weak statelessness puts the responsibility for state storage onto block proposers, while all other nodes on the network verify blocks without storing the full state data.
+
+**In weak statelessness proposing blocks requires access to full state data but verifying blocks requires no state data**
+
+For this to happen, [Verkle trees](../verkle-trees) must already have been implemented in Ethereum clients. Verkle trees are a replacement data structure for storing Ethereum state data that allow small, fixed size "witnesses" to the data to be passed between peers and used to verify blocks instead of verifying blocks against local databases. [Proposer-builder separation](../pbs) is also required because this allows block builders to be specialized nodes with more powerful hardware, and those are the ones that require access to the full state data.
+
+<ExpandableCard title="Why is it OK to rely on fewer block proposers?">
+
+Statelessness relies oin block builders maintaining a copy of the full state data so that they can generate witnesses that can be used to verify the block. Other nodes do not need access to the state data, all the information required to verify the block is available in the witness. This creates a situation where proposing a block is expensive, but verifying the block is cheap, which implies fewer operators will run a block proposing node. However, decentralization of block proposers is not critical as long as as many participants as possible can independently verify that the blocks they propose are valid.
+
+Read more on [Dankrad's notes](https://notes.ethereum.org/WUUUXBKWQXORxpFMlLWy-w#So-why-is-it-ok-to-have-expensive-proposers)
+
+</ExpandableCard>
+
+Block proposers use the state data to create "witnesses" - the minimal set of data that prove the values of the state that are being changed by the transactions in a block. Other validators do not hold the state, they only store the state root (a hash of the entire state). They receive a block and a witness and use them to update their state root. This makes a validating node extremely lightweight.
+
+Weak statelessness is in an advanced state of research, but it relies upon [Proposer-builder separation](../pbs) and Verkle Trees to have been implemented so that small witnesses can be passed between peers. This means weak statelessness is probably a few years away from Ethereum Mainnet.
```

Comment:
We already link to PBS just above
```suggestion
Weak statelessness is in an advanced state of research, but it relies upon proposer-builder separation and Verkle Trees to have been implemented so that small witnesses can be passed between peers. This means weak statelessness is probably a few years away from Ethereum Mainnet.
```